components/expenses/graphql/fragments.ts

```diff
@@ -2,6 +2,24 @@ import { gql } from '@apollo/client';
 
 import { collectiveNavbarFieldsFragment } from '../../collective-page/graphql/fragments';
 
+export const commentFieldsFragment = gql`
```

Comment:
Copy-paste? Shouldn't we remove the original one and make sure there's no duplicate `commentFieldsFragment`?